charts/actions-runner-controller/templates/ci-secret.yaml

```diff
@@ -7,4 +7,8 @@ data:
 kind: Secret
 metadata:
   name: controller-manager
+  {{- if .Values.authSecret.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Comment:
I believe you forgot to add `{{ with }}` here so `.` in L12 points to the root template data rather than the annotations object

ANd this can be slimmed down to one line as https://github.com/actions-runner-controller/actions-runner-controller/pull/824/files#r710589956

Reply:
updated. but need to test this one. will do it tomorrow